web/src/lib/components/album-page/albums-controls.svelte

```diff
@@ -88,12 +89,33 @@
   }
 
   $: sortIcon = $albumViewSettings.sortOrder === SortOrder.Desc ? mdiArrowDownThin : mdiArrowUpThin;
+
+  const albumFilterNames: { [key in AlbumFilter]: string } = {
+    [AlbumFilter.All]: $t('all'),
+    [AlbumFilter.Owned]: $t('owned'),
+    [AlbumFilter.Shared]: $t('shared'),
+  };
+
+  const albumSortByNames: { [key in AlbumSortBy]: string } = {
+    [AlbumSortBy.Title]: $t('sort_title'),
+    [AlbumSortBy.ItemCount]: $t('sort_items'),
+    [AlbumSortBy.DateModified]: $t('sort_modified'),
+    [AlbumSortBy.DateCreated]: $t('sort_created'),
+    [AlbumSortBy.MostRecentPhoto]: $t('sort_recent'),
+    [AlbumSortBy.OldestPhoto]: $t('sort_oldest'),
+  };
+
+  const albumGroupByNames: { [key in AlbumGroupBy]: string } = {
+    [AlbumGroupBy.None]: $t('group_no'),
+    [AlbumGroupBy.Owner]: $t('group_owner'),
+    [AlbumGroupBy.Year]: $t('group_year'),
+  };
 </script>
 
 <!-- Filter Albums by Sharing Status (All, Owned, Shared) -->
 <div class="hidden xl:block h-10">
   <GroupTab
-    filters={Object.keys(AlbumFilter)}
+    filters={Object.values(albumFilterNames)}
```

Comment:
Does this work for languages other than English? And does it remember the correct state after switching languages?

Reply:
Yes, it works for others, unfortunatelly it doesn't keep setting after language is switched, it saves dropdown text value, it should be fixed.

Reply:
Just tested it using development language and the buttons don't work for me. Remembering the selected state is also broken. This could be fixed by passing both the enum value and a translated label to the `GroupTab` component, and then only storing the enum as selected value.

Reply:
Fixed